components/FormCreate.js

```diff
@@ -73,9 +99,17 @@ export default function FormCreate() {
     (imageSearch && imageSearch.hits && imageSearch.hits.length > 0) &
     (searchTerm.length > 0);
 
+  // these solution is for the moment, we will build a Modal solution next week
+  function showAlert(event) {
+    event.preventDefault();
+    window.alert(
+      "Please ensure that both longitude and latitude values are selected before proceeding. You can easily select them by clicking on the 'Select Geodata' button."
+    );
+  }
+
   return (
     <>
-      <Form onSubmit={handleSubmit}>
+      <Form onSubmit={lat !== null ? handleSubmit : showAlert}>
```

Comment:
<Form onSubmit={lat !== null ? handleSubmit : showAlert}> according to this only lat is checked for null , how about lng?